adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/providers/LoraProvider.java

```diff
@@ -37,11 +37,20 @@
 
     /**
      * The url path prefix which is used for this provider. E.g. "/myloraprovider".
-     * 
+     *
      * @return The url path prefix with leading slash. E.g. "/myloraprovider".
      */
     String pathPrefix();
 
+    /**
+     * The content type which identifies content of this provider. E.g. "application/json+myloraprovider".
+     *
+     * @return The content type of this LoRaWAN provider. E.g. "application/json+myloraprovider".
+     */
+    default String getContentType(){
+        return HttpUtils.CONTENT_TYPE_JSON + "+" + getProviderName();
```

Comment:
I think we should try to adhere to common practices when [defining media types](https://tools.ietf.org/html/rfc6838). 
In particular, we should use a type defined in the *Vendor* tree, e.g. using `application/vnd.eclipse-hono` as the prefix. The suffix (the stuff after the `+` sign) should be a (registered) *Structured Syntax Name*.  Most likely `json` in our case.

We could then either add the provider name to the sub type name, e.g. 

`application/vnd.eclipse-hono.lora.actility+json`

or use a generic media type indicating that the data structure has been created by the LoRa protocol adapter, e.g.

`application/vnd.eclipse-hono.lora+json`

and then add the network provider's name as an application property to the AMQP message.

Personally, I think the latter one makes more sense, in particular if we want to also normalize a lot of the data.

Reply:
I've changed it again.
As for your suggestion on the application property of the AMQP message, I've seen that this is already done in the "customizeDownstreamMessage" function:
```
@Override
    protected void customizeDownstreamMessage(final Message downstreamMessage, final RoutingContext ctx) {
        MessageHelper.addProperty(downstreamMessage, LoraConstants.APP_PROPERTY_ORIG_LORA_PROVIDER,
                ctx.get(LoraConstants.APP_PROPERTY_ORIG_LORA_PROVIDER));
    }
```
The context variable is set earlier on using
`ctx.put(LoraConstants.APP_PROPERTY_ORIG_LORA_PROVIDER, provider.getProviderName());`